### PR TITLE
[Layer5-ng] feat. /learn/service-mesh

### PR DESCRIPTION
**Pull request by vineethvanga18**
Signed-off-by: vineethvanga18 <reddy.8@iitj.ac.in>

**Description**
Added katacoda section in '/learn'

**Notes for Reviewers**
Currently not fetching courses from katacoda but manually entering them(not sure if they have an API for that).

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Jashpatel1** [comment]
Are we going to have direct links @leecalcote ?

File: src/pages/learn/service-mesh.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import Navigation from "../../sections/Navigation";
+import Footer from "../../sections/Footer";
+import KatacodaServiceMesh from "../../sections/Katacoda-Service-Mesh";
+
+import { GlobalStyle } from "../../sections/app.style";
+import theme from "../../theme/blog/themeStyles";
+
+
+
+const ServiceMeshKatakodaPage = ({hide_path}) => {
```
Comment (Tanuj22) [inline src/pages/learn/service-mesh.js:16]: Why this Prop?

File: src/sections/Katacoda-Service-Mesh/Card.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import combined from "./combined.svg";
+import { Row, Col } from "../../reusecore/Layout";
+
+const Card = ({ title, info, link }) => {
```
Comment (Tanuj22) [inline src/sections/Katacoda-Service-Mesh/Card.js:5]: Give a better name and keep in `components` directory

File: src/sections/Katacoda-Service-Mesh/KatacodaWrapper.style.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+        h1{
+            margin-top: 1rem;
+            color: ${props => props.theme.black};
+        }
+        p{
+            margin: auto;
+            margin-top: 1rem;
+            color: ${props => props.theme.black};
+        }
+        button{
```
Comment (Tanuj22) [inline src/sections/Katacoda-Service-Mesh/KatacodaWrapper.style.js:64]: Don't change the color. To be done centrally when design approved

File: src/sections/Katacoda-Service-Mesh/KatacodaWrapper.style.js [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+        padding: 1rem 0;
+    }
+    h4 {
+        font-weight: bold;
+        font-size: 1.5rem;
+        line-height: 1.5rem;
+    }
+    p {
+        font-weight: normal;
+        font-size: 1.125rem;
```
Comment (Tanuj22) [inline src/sections/Katacoda-Service-Mesh/KatacodaWrapper.style.js:42]: Why a custom font size?

File: src/sections/Katacoda-Service-Mesh/KatacodaWrapper.style.js [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+        height: 100%;
+    }
+    .description {
+        display: flex;
+        flex-direction: column;
+        padding: 1rem 0;
+    }
+    h4 {
+        font-weight: bold;
+        font-size: 1.5rem;
```
Comment (Tanuj22) [inline src/sections/Katacoda-Service-Mesh/KatacodaWrapper.style.js:37]: Why a custom font size?

**Review by Tanuj22 — COMMENTED** [review]
We will probably also want a section for this in `/learn` @leecalcote?

File: src/pages/learn/service-mesh.js [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+
+import Layout from "../../components/layout";
+import SEO from "../../components/seo";
+
+import Navigation from "../../sections/Navigation";
+import Footer from "../../sections/Footer";
+import KatacodaServiceMesh from "../../sections/Katacoda-Service-Mesh";
+
+import { GlobalStyle } from "../../sections/app.style";
+import theme from "../../theme/blog/themeStyles";
```
Comment (Tanuj22) [inline src/pages/learn/service-mesh.js:12]:
```suggestion
import theme from "../../theme/app/themeStyles";
```

**Comment by leecalcote** [comment]
@Jashpatel1, question question. Ideally, not. We want to keep users on our site as much as possible. Instead of sending them to katacoda.com, we can embed the labs like what has been done here – https://docs.meshery.io/service-meshes/adapters/consul

**Comment by vineethvanga18** [comment]
@leecalcote Embedding individual courses is not yet done. I didn't find the designs in figma (How do we list individual scenarios and how to embed these scenarios)

**Comment by leecalcote** [comment]
Ooooh, @Neil-140301... 😄

**Review by leecalcote — APPROVED** [review]
Oh! I see now. Thanks, @vineethvanga18

**Comment by leecalcote** [comment]
Dang it. This was the wrong PR. _sigh_

**Pull request merged**